helm-charts/chatqna/values.yaml

```diff
@@ -6,6 +6,8 @@
 # Declare variables to be passed into your templates.
 
 replicaCount: 1
+horizontalPodAutoscaler:
```

Comment:
HPA options in all the values files could include following comment:
```suggestion
# Enabling HPA will:
# - Ignore above replica count
# - Overwrite existing PrometheusAdapter "adapter-config" configMap (with ChatQnA specific custom metric queries):
#   https://github.com/kubernetes-sigs/prometheus-adapter/blob/master/deploy/manifests/config-map.yaml
horizontalPodAutoscaler:
```

Reply:
Why drop this comment?
`# - Ignore above replica count, as it will be controlled by HPA`

Reply:
Unintentional, rebase / merge-conflict-resolve artifact probably.

Reply:
Ah, I see what you mean. There is no HPA for ChatQnA service itself, only for tgi, embedding and reranking. TGI is not this service, this is an app service, tgi is in common/tgi.